test/plugins/test_definitions.py

```diff
@@ -40,6 +55,40 @@ def test_definitions(config, workspace):
     )
 
 
+def test_indirect_definitions(config, workspace):
+    # Over 'subscripted_before_reference'
+    cursor_pos = {"line": 16, "character": 0}
+
+    # The definition of 'subscripted_before_reference',
+    # skipping intermediate writes to the most recent definition
+    def_range = {
+        "start": {"line": 14, "character": 0},
+        "end": {"line": 14, "character": len("subscripted_before_reference")},
+    }
+
+    doc = Document(DOC_URI, workspace, DOC)
+    assert [{"uri": DOC_URI, "range": def_range}] == pylsp_definitions(
+        config, doc, cursor_pos
+    )
+
+
+def test_definition_with_multihop_inference_goto(config, workspace):
+    # Over 'subscripted_before_reference'
```

Comment:
```suggestion
    # Over 'inception()'
```

Reply:
Woops thanks for catching that! Fixed now